services/brig/src/Brig/API/Public.hs

```diff
@@ -1115,13 +1090,12 @@ customerExtensionCheckBlockedDomains email = do
         when (domain `elem` blockedDomains) $
           throwM $ customerExtensionBlockedDomain domain
 
-createConnectionH :: JSON ::: UserId ::: ConnId ::: JsonRequest Public.ConnectionRequest -> Handler Response
-createConnectionH (_ ::: self ::: conn ::: req) = do
-  cr <- parseJsonBody req
+createConnectionH :: UserId -> ConnId -> Public.ConnectionRequest -> Handler (Public.ResponseForExistedCreated Public.UserConnection)
+createConnectionH self conn cr = do
```

Comment:
Perhaps this is just nitpicking, but I got the impression that the naming convention where `H` is appended to handler names applies only to old `wai-route` endpoint handlers.

Reply:
Makes sense 👍🏿